downstream/assemblies/hub/assembly-high-availability-hub-selinux.adoc

```diff
@@ -0,0 +1,83 @@
+[id="assembly-high-availability-hub-selinux"]
+
+= Setting up a high availability automation hub on SELinux
+
+To set up a high availability (HA) deployment of {HubName} on SELinux, you will need to create two mount points, one for `/var/lib/pulp` and one for `/var/lib/pulp/pulpcore_static`, before you run the installer.
+
+.Pre-installation procedure
+. Create a mount point at `/var/lib/pulp`:
++
+----
+$ mkdir -p /var/lib/pulp/
+----
+. Open `/etc/fstab` using a text editor, then add the following values:
++
+----
+srv_rhel8:/data /var/lib/pulp nfs defaults,_netdev 0 0
+srv_rhel8:/data/pulpcore_static /var/lib/pulp/pulpcore_static nfs defaults,_netdev,context="system_u:object_r:httpd_sys_content_rw_t:s0" 0 0
+----
+. Run the mount command:
++
+----
+$ mount -a
+----
```

Comment:
So they actually need to run:
```
mount /var/lib/pulp
mkdir /var/lib/pulp/pulpcore_static
mount /var/lib/pulp/pulpcore_static
```
Because the folder needs to exist, and it cannot be created until after the 1st is mounted.